app/1.0/docs/devguide/data-binding.md

```diff
@@ -59,18 +91,21 @@ For example:
 syntax. For more information, see [Binding to native element attributes](#native-binding).
 { .alert .alert-info }
 
+If the property being bound is an object or an array, both elements get a reference to the **same
+object**. This means that either element can change the object and a true one-way binding is
+impossible. For more information, see [Data flow for objects and
```

Comment:
I haven't tried it but I wonder if this is true if the object was frozen using [Object.freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze)